scripts/docs.angularjs.org-firebase/functions/.eslintrc.json

```diff
@@ -0,0 +1,123 @@
+{
+  "parserOptions": {
+    // Required for certain syntax usages
+    "ecmaVersion": 6
+  },
+  "plugins": [
+    "promise"
+  ],
+  "extends": "eslint:recommended",
```

Comment:
Why do we not use our rules but extend the recommended preset?

Reply:
These are auto-generated by Firebase and I didn't bother to review them, tbh

Can we extend our rules / preset and still allow ES6 stuff? Because we don't allow that for our scripting stuff.

Reply:
We should allow ES6 for our Node.js stuff in general now that we run the build in Node 8.

Is just the ESLint config generated by Firebase or the whole code in this directory? If the latter then just add `"root": true` to `.eslintrc.json` so that it doesn't inherit our config.